convert2rhel/checks.py

```diff
@@ -705,118 +705,117 @@ def check_package_updates():
 
 def is_loaded_kernel_latest():
     """Check if the loaded kernel is behind or of the same version as in yum repos."""
-    logger.task("Prepare: Check if the loaded kernel version is the most recent")
-
+    logger.task("Prepare: Checking if the loaded kernel version is the most recent")
```

Comment:
You are reverting already merged changes from at least https://github.com/oamg/convert2rhel/pull/726 and https://github.com/oamg/convert2rhel/pull/687. Why's that?

Reply:
Hmmm I had did a rebase in the PR to go back to how it was on my first commit, as when I did a rebase with main I think some code that I didn't edit changed. so I wanted to go back to how the PR was before I tried to rebase. 

I am working on the fixes for this now and will push the changes.